aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapterDeleteTest.java

```diff
@@ -301,4 +304,43 @@ public void deleteNoneDoesNotFail() throws DataStoreException {
         final List<BlogOwner> blogOwners = adapter.query(BlogOwner.class);
         assertEquals(Collections.singletonList(john), blogOwners);
     }
+
+    /**
+     * Test deleting item with many children does not trigger SQLite exception.
+     * SQLite places a hard limit of 1000 on expression tree depth, so chaining too many
```

Comment:
Is this limit exposed with a constant by any of the SQL APIs? `andorid.sql.SqlGizmo.SOME_LIMIT`, that kind of thing?

Reply:
i believe this variable lives internally in sqlite3, so it isn't directly accessible from android code.